packages/animations/lib/src/open_container.dart

```diff
@@ -235,8 +261,9 @@ class _OpenContainerState extends State<OpenContainer> {
   final GlobalKey _closedBuilderKey = GlobalKey();
 
   Future<void> openContainer() async {
-    await Navigator.of(context, rootNavigator: widget.useRootNavigator)
-        .push(_OpenContainerRoute(
+    final T data =
+        await Navigator.of(context, rootNavigator: widget.useRootNavigator)
+            .push(_OpenContainerRoute<T>(
```

Comment:
nit: Add a trailing comma, it would help with formatting this block:
```suggestion
    final T data = await Navigator.of(
      context, 
      rootNavigator: widget.useRootNavigator,
    ).push(_OpenContainerRoute<T>(
```